pymc/distributions/multivariate.py

```diff
@@ -2706,7 +2702,12 @@ def check_zerosum_axes(cls, n_zerosum_axes: Optional[int]) -> int:
 
     @classmethod
     def rv_op(cls, sigma, n_zerosum_axes, support_shape, size=None):
-        shape = to_tuple(size) + tuple(support_shape)
+        if size is not None:
+            shape = tuple(size) + tuple(support_shape)
```

Comment:
Do we need to check that `size` and `sigma.shape` are compatible or that done somewhere else before this call?

Reply:
We shouldn't have to check. Shape issues will arise when creating `Normal` below or at evaluation time when broadcasting fails between size and sigma